websites/M/Minecraft Wiki/presence.ts

```diff
@@ -38,27 +37,29 @@ presence.on("UpdateData", async () => {
 			btnViewThread: "apple.btnViewThread",
 			viewWatchlist: "minecraft wiki.viewWatchlist",
 		}),
+		mwConfig = await presence.getPageletiable<{
```

Comment:
Preferably, use `getPageVariable` which has a better performance and is not deprecated.

Reply:
I was looking at https://docs.premid.app/en/dev/presence/class which didn't list that function being deprecated and didn't mention getPageVariable at all. How would I use that function correctly? Can I use `mw.config.get(['name','name'])` instead of `mw.config.values`?

Reply:
Yeah, the documentation is in the process of being updated. Usage looks like this:
```ts
const {
  "mw.config.values.foo": foo,
  "mw.config.values.bar": bar,
} = presence.getPageVariable([
  "mw.config.values.foo",
  "mw.config.values.bar"
]);
```

Reply:
Hm, I would prefer to have the variables on a single object variable instead of them each being their own variable.

Reply:
I've switched to getPageVariable